### PR TITLE
Updated Readme and added AMP.md to split technical info and avoid ove…

### PR DESCRIPTION
**Pull request by gaetanmarmasse**
…rloading the main readme

| Q                        | A 
| ------------------------ | ---
| Fixed Issues?            | N/A
| Patch: Bug Fix?          | No
| Minor: New Feature?      | No
| Major: Breaking Change?  | No
| Tests Added + Pass?      | Yes
| Documentation Provided   | Yes (code comments and or markdown)
| Any Dependency Changes?  | No
| License                  | Apache License, Version 2.0

This PR adds basic documentation in split README files.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,6 +69,24 @@ You can also subscribe via Email [aem-core-components-dev+subscribe@googlegroups
 
 Visit the [roadmap wiki page](https://github.com/adobe/aem-core-wcm-components/wiki) for the main upcoming components and features.
 
+## Accelerated Mobile Pages support
+
+Google's AMP suport has been added to the AEM Core Components. Learn more about Accelerated Mobile Pages [here](https://amp.dev/).
+
+The [Component Library](https://www.adobe.com/go/aem_cmp_library) and [its code](examples/src/content/jcr_root/apps/core-components-examples) in this repository show how it can be used. 
+
+###AMP behavior control
```
Comment (paigekeenan) [inline README.md]: needs a space between `###AMP` - It's currently displaying the hashes in the output.

File: AMP.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+#Accelerated Mobile Pages.
```
Comment (paigekeenan) [inline AMP.md]: The readMe is currently outputting the `#` with all of the headlines. They all need a space between the last `#` and the headline text. (Click the "Display the rich diff" button above to view this.)

File: AMP.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+Learn more about how AMP mode can be cotrolled [here](README.md)
+
+##Major Technical aspects
+###Parallel delivery
+The promise of using AMP in AEM is to not rewrite content but to be  able to serve the usual HTML pages along with their AMP versions if desired. To support this, AEM core components are relying on the `amp` sling selctor to render a `page.html` and `page.amp.html` if requested. 
+
+A [Transformer](bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/services/amp/AmpTransformer.java) is used to also inject the proper `<link>` in the head of the document to inform an HTML page has an AMP "sibling" and vice-versa. This allows for AMP discoverability of AEM pages.
+
+###Inline CSS
+AMP requires all the CSS used for a page to be inline in the `<head>` element of the document. This also has to be **optimized** so that the CSS code does not represent all possible component combination but only the ones needed for the page currently being rendered.
```
Comment (paigekeenan) [inline AMP.md]: `#nitpick` combination should be plural

**Review by paigekeenan — APPROVED** [review]
I have a couple call outs but overall think this is an informative representation of the work we've done. I'd be good with merging once they are addressed. LGTM 🎉

File: AMP.md (hunk 3 above)
Comment (gaetanmarmasse) [reply AMP.md]: 👍

File: AMP.md (hunk 2 above)
Comment (gaetanmarmasse) [reply AMP.md]: Fixed

File: README.md (hunk 1 above)
Comment (gaetanmarmasse) [reply README.md]: Fixed

**Comment by gaetanmarmasse** [comment]
Comments addressed, Thanks @paigekeenan .

**Pull request merged**